xml/art_amd-sev.xml

```diff
@@ -118,7 +118,9 @@
    All <literal>virtio-net</literal> devices need to be configured with the iPXE
```

Comment:
not sure if mentioned elsewhere, but should we mention that all devices (not only virtio-net) should have option roms disabled?

Reply:
Interesting. If that is the case then probably this will need to be adjusted in KubeVirt as well. So is it confirmed?

Reply:
I just tried e1000 and it was NOT necessary to disable any option ROM.

Reply:
IDK. Maybe @joergroedel or someone familiar with virtio devices.